### PR TITLE
Feature: Add log file for TUI

### LINKED ISSUE #32: Add log exports from tui

### PR DESCRIPTION
**Pull request by sambukowski**
Because the TUI manipulates the terminal output, a dedicated log file is required to capture the logs of the app that would normally be written to stdout of stderr.

closes #32

### CODE REVIEW

File: cmd/devtools/run.go [hunk 1]
```diff
@@ -57,6 +57,19 @@ func runCmdHandler(c *cobra.Command, args []string) {
 	instance := c.Flag("instance").Value.String()
 	IsInstanceNameValidOrPanic(instance)
 
+	// create log file for the UI app
+	logPath := filepath.Join(astriaDir, instance, "ui.log")
```
Comment (steezeburger) [inline cmd/devtools/run.go:61]:
```suggestion
	logPath := filepath.Join(astriaDir, instance, "astria-go.log")
```

File: cmd/devtools/run.go [hunk 2]
```diff
@@ -57,6 +57,19 @@ func runCmdHandler(c *cobra.Command, args []string) {
 	instance := c.Flag("instance").Value.String()
 	IsInstanceNameValidOrPanic(instance)
 
+	// create log file for the UI app
```
Comment (steezeburger) [inline cmd/devtools/run.go:60]: We've got the `cmd/logging.go` file. Can this logic go in a public function that lives there?

File: README.md [hunk 3]
```diff
@@ -237,6 +238,20 @@ go install github.com/spf13/cobra-cli@latest
...
+### TUI Logs
+
+Because the TUI that is launched when using `dev run` manipulates the terminal,
+the logs that would usually go to `stdout` or `stderr` are written to a log
+file. You can find this log file in the `~/.astria/<instance>` directory that
+the CLI is using for `dev run`.
+For example, if you run `astria-go dev run`, the log file for the
+currently running TUI will be at `~/.astria/default/ui.log`.
+
+> NOTE: This log file gets overwritten every time you run the TUI. If you need
```
Comment (steezeburger) [inline README.md:250]: How does this file behave if you were to, e.g.
* open terminal window and `dev run`
* open terminal window and `tail -f ~/.astria/default/astria-go.log`
* quit tui
* rerun `dev run`

**Review by steezeburger — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sambukowski) [reply README.md:250]: Tailing the log is unaffected by restarting the tui. The window running `tail` will show all the logs continuously across restarts/runs, but the `astria-go.log` will only show the logs for the current ui session.

**Review by steezeburger — APPROVED** [review]

**Pull request merged**